src/shell.js

```diff
@@ -220,7 +221,7 @@ else if (ENVIRONMENT_IS_WEB || ENVIRONMENT_IS_WORKER) {
         return new Uint8Array(xhr.response);
 #if SUPPORT_BASE64_EMBEDDING
       } catch (err) {
-        var data = tryParseAsDataURI(f);
+        var data = tryParseAsDataURI(url);
```

Comment:
this looks correct, but it suggests we don't have a test for this code path. @buu700, can you please look into that?

Reply:
Oops, sorry about the typo there. As far as the code path, it looks like to test this we would need to be combining SINGLE_FILE and/or mem init mode 0 with a Worker, and the only SINGLE_FILE-related browser test we have now is to make sure it still works without a native `atob` implementation.

Reply:
Ahh, I just spent way too long trying to figure out what code path was hitting this. It looks like there isn't actually a code path that hits this line; the test @nazar-pc fixed must've been from Closure Compiler doing some static analysis on the output.

If that's the case, the test that failed for him is probably already the best test we're gonna get without refactoring and writing a bunch of unit tests (and/or incrementally adding more static analysis from Closure/TypeScript/Flow).

Reply:
Yes, Closure Compiler was complaining about the variable being undefined

Reply:
Got it, thanks.